tests/python_functional/src/message_reader/message_reader.py

```diff
@@ -20,9 +20,8 @@
 # COPYING for details.
 #
 #############################################################################
-
-from src.common.blocking import wait_until_true_custom
 from src.common.blocking import DEFAULT_TIMEOUT
+from src.common.blocking import wait_until_true_custom
```

Comment:
What is the idea here? In `test_message_reader.py` the order is the opposite (`READ_ALL_MESSAGES` vs `MessageReader`)

Reply:
The rule is: they are in alphabetical order